babel.config.js

```diff
@@ -170,7 +170,7 @@ module.exports = function (api) {
       // presets are applied from right to left
       ["@babel/env", envOpts],
       [
-        "@babel/preset-typescript",
+        "babel-preset-stable-typescript",
```

Comment:
Originally I wanted to use `@babel-stable/preset-typescript`, but it doesn't work.

Reply:
I'm confused 😅 
Why do we need to rename it here? Can't we just use a random name when testing locally the "current" version, and leave it here like this?

Reply:
https://github.com/babel/babel/pull/15478/files#diff-87d5fe6dc73b89f7f5c51127c02597480d8d5ef96d55be38bc08e9fbf12bf059R151

It's in the release code.

Reply:
Oh, you could try adding the preset from the workspace as a dev dependency of `@babel/core` instead.